### PR TITLE
Improved schema.

### LINKED ISSUE #14: DB - Remove guards and related queries
We are currently using "guards" table, we've already decided we want to use "roles" on the users table rather than adding another one.

### PR DESCRIPTION
**Pull request by inbarbarkai**

### CODE REVIEW

File: db_init/baseline.sql [hunk 1]
```diff
@@ -1,43 +1,51 @@
...
-	creation_time TIMESTAMPTZ,
-	mod_time TIMESTAMPTZ,
-	comment TEXT,
+	creation_timestamp TIMESTAMPTZ,
+	modify_timestamp TIMESTAMPTZ,
+	comment_for_guard TEXT,
 	PRIMARY KEY (invitation_id)
 );
 
 create table guards(
```
Comment (yarons) [inline db_init/baseline.sql]: @inbarbarkai Do you want to remove it before the PR or we'll fix it afterwards?

**Comment by dangalg** [comment]
Can I suggest "user" instead of "summoner" in events table, in order to keep it cohesive?

**Comment by yarons** [comment]
Yes, we should also update the wiki accordingly, feel free to hoard on Inbar's branch and suggest it yourself (if you add changes to another contributor's branch it's being added to the PR automatically).

File: db_init/baseline.sql [hunk 2]
```diff
@@ -1,43 +1,51 @@
...
-	event_time TIMESTAMPTZ NOT NULL,
+	event_id UUID NOT NULL,
+	event_timestamp TIMESTAMPTZ NOT NULL,
 	event_type event_type NOT NULL,
 	amount_before INT,
 	amount_after INT,
-	summoner VARCHAR(100),
-	guard_id INT REFERENCES guards (guard_id),
-	invitation_id VARCHAR(16)
+	summoner UUID REFERENCES users (user_id),
```
Comment (yarons) [inline db_init/baseline.sql]: Suggested changing to "User", I think it's a good idea, @inbarbarkai ?

Comment (dangalg) [inline db_init/baseline.sql]:
```suggestion
	user_id UUID REFERENCES users (user_id),

```

**Review by dangalg — COMMENTED** [review]
user instead of summoner, for cohesiveness.

**Review by yarons — APPROVED** [review]
Awesome ERD, thank you!

**Pull request merged**